backend/src/test/java/ch/puzzle/pcts/mapper/CalculationMapperTest.java

```diff
@@ -68,25 +78,60 @@ private Role createRole() {
     }
 
     private Calculation createCalculation(Member member, Role role) {
-        return new Calculation(CALC_ID, member, role, STATE, PUBLICATION_DATE, PUBLICIZED_BY);
+        Calculation calculation = Calculation.Builder
+                .builder()
+                .withId(CALC_ID)
+                .withMember(member)
+                .withRole(role)
+                .withState(STATE)
+                .withPublicationDate(PUBLICATION_DATE)
+                .withPublicizedBy(PUBLICIZED_BY)
+                .withDegrees(List.of())
+                .withExperiences(List.of())
+                .withCertificates(List.of())
+                .build();
+        calculation.setPoints(POINTS);
+        return calculation;
     }
 
-    private CalculationInputDto createInput() {
-        return new CalculationInputDto(MEMBER_ID, ROLE_ID, STATE);
+    private CalculationInputDto createCalculationInput() {
+        DegreeCalculationInputDto degreeInput = mock(DegreeCalculationInputDto.class);
+        ExperienceCalculationInputDto experienceInput = mock(ExperienceCalculationInputDto.class);
+        return new CalculationInputDto(MEMBER_ID,
+                                       STATE,
+                                       ROLE_ID,
+                                       List.of(CERTIFICATE1_ID, CERTIFICATE2_ID),
+                                       List.of(LEADERSHIP_EXPERIENCE_ID),
+                                       List.of(degreeInput),
+                                       List.of(experienceInput));
+    }
+
+    private MemberDto mockMemberDto() {
+        return mock(MemberDto.class);
+    }
+    private RoleDto mockRoleDto() {
+        return mock(RoleDto.class);
     }
 
+    // ---------------- Tests ----------------
+
     @DisplayName("Should map Calculation to CalculationDto")
     @Test
     void shouldReturnCalculationDto() {
```

Comment:
create parametrized tests for every test in this file to ensure things gets always mapped properly 
instead of just checking the optimal case